cwltool/main.py

```diff
@@ -8,9 +8,12 @@
 import io
 import logging
 import os
+import parser
```

Comment:
I think is not needed @jjkoehorst ?

I cannot see `parser` being used here in the GitHub UI diff. If I open the complete module, I can see a `parser` variable being instantiated, but nothing imported - https://github.com/common-workflow-language/cwltool/blob/bd61e436150c87af758061130a9c6650a0918ea0/cwltool/main.py#L1022

After removing this line the `cwltool` command works with no issues.